apps/dashboard/config/configuration_singleton.rb

```diff
@@ -387,6 +387,14 @@ def open_apps_in_new_window?
     end
   end
 
+  def show_bc_card_memory?
+    if ENV['OOD_BC_CARD_MEMORY']
+      to_bool(ENV['OOD_BC_CARD_MEMORY'])
+    else
+      false
+    end
+  end
```

Comment:
I don't think we need a config for this, it's just another thing us  to document and keep track for the admin to miss.